addon/services/cloudinary.js

```diff
@@ -181,5 +181,97 @@ export default Ember.Service.extend({
     }
 
     return `${protocol}${subDomain}.${domain}${pathname}`;
+  },
+
+
+  /** @property {String[]} concatenatedTransforms */
+  concatenatedTransforms: computed('config.CONCATENATED_TRANSFORMS', function() {
```

Comment:
`concatenatedTransforms`  is better have prefix of `_` so it will be understood it's private. In the helper it was unneeded since, you almost never use `Helper` class directly